content/en/blog/2023/ambient-merged-istio-main/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Istio Ambient Service Mesh Merged to Istio’s Main Branch"
+description: A significant milestone for ambient mesh
+publishdate: 2023-02-27
+attribution: "John Howard (Google), Lin Sun (Solo.io)"
+keywords: [istio,ambient]
+---
+
+Istio ambient service mesh was [launched in Sept 2022](/blog/2022/introducing-ambient-mesh/) in an experimental branch, introducing a new data plane mode for Istio without sidecars. Through collaboration with the Istio community, across Google, Solo.io, Microsoft, Intel, Aviatrix, Huawei, IBM and others, we are excited to announce that Istio ambient mesh has graduated from the experimental branch and merged to Istio’s main branch! This is a significant milestone for ambient mesh, paving the way for releasing ambient in Istio 1.18 and becoming the default in Istio’s future releases.
+
+## Major Changes from the Initial Launch
+
+Ambient mesh is designed for simplified operations, broader application compatibility, and reduced infrastructure cost. The ultimate goal of ambient is transparent to your applications and we have made a few changes to make ztunnel and waypoint components simpler and lightweight.
+
+* The ztunnel component has been rewritten from the ground up to be fast, secure, and lightweight. Refer to the “Introducing Rust-Based Ztunnel for Istio Ambient Service Mesh” blog for more information.
```

Comment:
creating this blog PR shortly, will update the link

Reply:
I think this will have to be merged first before the link can be updated, else lint would fail.  PR is created https://github.com/istio/istio.io/pull/12806

Reply:
now that the other PR is merged, link is updated. cc @frankbu